resources/detailsview/main.js

```diff
@@ -26,16 +26,6 @@
         });
     }
 
-    // Handle messages sent from the extension to the webview
-    // window.addEventListener("message", (event) => {
-    //     const message = event.data; // The json data that the extension sent
-    //     switch (message.type) {
-    //         case "searchResult": {
-    //             break;
-    //         }
-    //     }
-    // });
-
```

Comment:
Found some commented out code, so figured you might not want that left in there.